### PR TITLE
#167733639 Setup API documentation using swagger

### PR DESCRIPTION
**Pull request by emmanuelnwankwo**
#### What does this PR do?
Setup API documentation using swagger
#### Description of Task to be completed?
Have the following endpoints working
`GET api/docs`
#### How should this be manually tested?
* Clone the repo and cd into the project folder
* Git checkout ch-setup-api-documentation-167733639
* npm install
* npm run start:dev
* Launch http://localhost:3000/api/docs on the browser

#### What are the relevant pivotal tracker stories?
[#167733639](https://www.pivotaltracker.com/story/show/167733639)
#### Screenshots
api_doc

### CODE REVIEW

**Review by dchima — COMMENTED** [review]
**in src/routes/api/index.js**
I suggest we change the module import convention to ES6, just for readability moving forward.

**Comment by emmanuelnwankwo** [comment]
@dchima Thanks for pointing it out

**Review by dchima — APPROVED** [review]
The Documentation works fine. 
Also good to see hound fishing for ES6. Once ESLint is configured and in use, it shouldn't bark at the const and imports.

**Comment by chis0m** [comment]
Looks good. Also, consider swagger-jsdoc. It makes documentation easy
https://www.npmjs.com/package/swagger-jsdoc

**Review by Baystef — APPROVED** [review]
Swagger docs works correctly on the specified route.

**Comment by King-Success** [comment]
Looks good, has my approval

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "@babel/preset-env",
+        {
+          "useBuiltIns": "entry"
+        }
+      ]
+    ],
+    "plugins": [
+      "@babel/plugin-transform-runtime"
+    ]
+  }
```
Comment (King-Success) [inline .babelrc]: This file is in need of a line break at the bottom

File: swagger.json [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+                "example": {
+                    "username": "Example",
+                    "email": "Example",
+                    "bio": "Example",
+                    "image": "Example",
+                    "password": "Example"
+                }
+            }
+        }
+    }
```
Comment (King-Success) [inline swagger.json]: This file also needs a line break at the bottom

**Review by King-Success — APPROVED** [review]

**Pull request merged**